.github/workflows/e2e-playwright-allure_report.yml

```diff
@@ -0,0 +1,104 @@
+name: Playwright E2E Tests
+on:
+  workflow_dispatch:
+     inputs:
+      project:
+        description: 'Select the Browser to be used for E2E Testing'
+        required: true
+        default: 'chromium_test_workflow'
+        type: choice
+        options:
+          - chromium_test_workflow
+          - firefox_test_workflow
+          - webkit_test_workflow
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  run-playwright-e2e:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+
+      - name: Add Environment Variable To DependencyTrack Apiserver
+        run: |
+          cat <<EOF > docker-compose.override.yml
+          services:
+            apiserver:
+              environment:
+                ALPINE_BCRYPT_ROUNDS: "4"
+          EOF
+
+      - name: Start DependencyTrack
+        run: |
+          docker compose -f docker-compose.yml -f docker-compose.override.yml --profile demo up -d --pull always
+
+      - name: Generate Random UUID for password use
+        run: |
+          RANDOM_PASSWORD=$(uuidgen -r)
+          echo "::add-mask::$RANDOM_PASSWORD" # Mask the value
+          echo "RANDOM_PASSWORD=$RANDOM_PASSWORD" >> $GITHUB_ENV
+
+      - name: Install necessary playwright dependencies
+        run: |
+          npm ci
+          npx playwright install --with-deps
+
+      - name: Generate BDD Tests
+        run:  npx bddgen
+
+      - name: Run Playwright tests
+        run: npx playwright test --project=${{ inputs.project || 'chromium_test_workflow' }}
+
+      - name: Upload test results
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: playwright-results
+          path: playwright-test-results/allure-results
+
+  publish-results:
+    needs: run-playwright-e2e
+    runs-on: ubuntu-latest
+    if: always()
```

Comment:
Let's limit this to only run on the official repository, so that it doesn't run in forks. Like so: https://github.com/DependencyTrack/hyades/blob/95e4d6a10cffa5f9d0dceadbda6b6a66b45b6294/.github/workflows/release.yml#L32